examples/playground/playground.js

```diff
@@ -0,0 +1,4 @@
+const { create, all } = require('../..')
+const math = create(all)
+
+console.log(math.distance([10, 2, 5, 13, 6, 12, 0, 16, 8], [1, 11, 14, 4, 15, 3, 9, 7, 17]))
```

Comment:
I think we want to remove this file before merging.

Reply:
oh yeah sorry I forgot about that, just been using it for testing quickly

Reply:
I like the idea of the playground. We do have something similar already: [a REPL and a CLI](https://mathjs.org/docs/command_line_interface.html):

```
cd mathjs
./bin/repl.js
```

At least this is what I regularly use myself. @Veeloxfire do you use this playground file via node.js, or do you load it in the browser? Would the REPL that's there already work for you?

Reply:
ah yes this is exactly what I was looking for!

Reply:
:+1: